src/polygon.typ

```diff
@@ -0,0 +1,59 @@
+/// Returns a list of polygon points from
+/// a list of segments.
+///
+/// Cubic segments get linearized by sampling.
+///
+/// - segment (list): List of segments
+/// - samples (int): Number of samples
+/// -> List of vectors
```

Comment:
- Whats a polygon point
- use `array` instead of `list`
- set the return type as `array` and describe them as an array of vectors in the description.